### DISCUSSION TITLE
PyInstaller poiting to wrong spec file

### DISCUSSION THREAD

**Discussion by krzysztofkrzysztof** [pyinstaller/pyinstaller#8276] [Help] [answered]
Hello!

My scenario in cmd is running pyinsteller with a spec file with a typo in a format:
>pyinstaller myFile.**`cpes`** 

got an info about file not found obviously

then fix format:
>pyinstaller myFile.**`spec`**

pyInstaller still telling me that: 
>script 'path\myFile.**`cpes`**.'  not found

Did anyone faced an issue like that?

>257 INFO: PyInstaller: 6.3.0
>257 INFO: Python: 3.12.1
>278 INFO: Platform: Windows-10-10.0.19044-SP0

Q: Did you tried to close and open it again?
A: For cmd console yes, for system no.

**Comment by rokm** [comment, marked as answer]
That's because when you made the typo in `pyinstaller myFile.cpes`, PyInstaller created new spec file called `myFile.spec` (overwriting any existing spec file with that name) that points to `myFile.cpes` as the entry-point script. Because technically, the entry-point script *could* have a name like that (i.e., it does not really have to have .py suffix, or have a suffix at all).

I suppose prior to (re)generating the spec file, we could check that the entry-point script actually exists to avoid issues like that.

**Comment by krzysztofkrzysztof** [comment]
Yes, you are right. I haven't noticed that console message at first time:
>1450 INFO: wrote path\myFile.spec

> I suppose prior to (re)generating the spec file, we could check that the entry-point script actually exists to avoid issues like that.

Edge case caused by user typo, but could be a good idea.
Many thanks!
